### ISSUE TITLE
Permission Install

### ISSUE THREAD

**Issue by nickkecooper** [nabeelio/phpvms#693]
When I go to install this it is showing the uploads folder has incorrect permissions. I double checked and it has full permissions. Any idea on this? All the other files check out just fine.

**Comment by nickkecooper** [comment]
Seems like I fixed it by creating a public folder and then created the uploads folder within that folder, inside the phpvms folder. I moved the whole public folder to the public_html directory as stated. I assumed the uploads folder needed to stay behind with the root files?

**Comment by nabeelio** [comment]
Yeah, the storage folder needs to stay where it is, if that's what you mean

**Comment by nabeelio** [comment]
Oh, I see what you mean. That might be a bug where it's still looking for that folder within the phpvms directory and not where public is currently. Lemme look at that. Did you move the `/public` contents out?

**Comment by nickkecooper** [comment]
I moved the whole /public folder in the the /public_html folder. Once I re-created a /public folder and added /uploads to that folder with correct permissions, the install went smoothly. Glad I can point stuff out I hope to be of help with this project!

**Comment by nabeelio** [comment]
No problem, lemme just clarify... so the index.php, etc, you moved into `public_html`, and then had to create a `public` folder in there with `/uploads` to get that to work? Did you still have an `/uploads` folder under `public_html`?

**Comment by nickkecooper** [comment]
Sorry, yes I moved the contents in the /public folder into the /public_html. Once I moved the contents I deleted the /public folder. The installer said I did not have the right permissions on the /public/upload folder. I realized that it does not exist in that location as I had moved it to the public_html. Once I created the /public and /uploads folder I was able to do the install. I hope its more clear now :)
